### PR TITLE
[WIP] Update changelog for next release

### PR DESCRIPTION
**Pull request by patrickjahns**
This PR so far includes everything I could find in the history and we can use it to prepare the next release. 

If the information is okay, I will backport it to stable10 branch

### CODE REVIEW

**Comment by PVince81** [comment]
seems many stable10 backport PRs were raised by @felixheidecke already.

Note that for 10.0.3RC1 I'd rather deliver what was already tested.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,44 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Added
 
 - Added market:uninstall command - [#125](https://github.com/owncloud/market/pull/125)
+- Added background job to notify admins about app updates - [#108](https://github.com/owncloud/market/pull/108)
+
+### Changed
+
+- `occ market:list` will return a alphabetical sorted list - [#122](https://github.com/owncloud/market/pull/112)
+- `occ market` commands will return non-zero exit codes on failure - [#143](https://github.com/owncloud/market/pull/143)\
```
Comment (PVince81) [inline CHANGELOG.md]: stray backslash

Comment (PVince81) [reply CHANGELOG.md]: this was already backported to stable10??

Comment (patrickjahns) [reply CHANGELOG.md]: yes https://github.com/owncloud/market/pull/147

**Comment by patrickjahns** [comment]
Changelog only includes changes that are now both in master/stable10 

Should I backport/copypaste it over to stable10?

**Comment by PVince81** [comment]
@patrickjahns I need a changelog on stable10 that only contains whatever changes exist on stable10, so do whatever is necessary. Backport, etc.

**Comment by PVince81** [comment]
please backport

**Pull request merged**